src/Faker/Provider/fr_CI/Person.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Faker\Provider\fr_CI;
+
+class Person extends \Faker\Provider\Person
+{
+    protected static $maleNameFormats = array(
+        '{{lastName}} {{firstNameMale}}',
+        '{{lastName}} {{firstNameMale}}',
+        '{{lastName}} {{firstNameMale}}',
+        '{{lastName}} {{firstNameMale}}',
+        '{{lastName}} {{firstNameMale}}',
+        '{{firstNameMale}} {{prefix}} {{lastName}}',
+        '{{lastName}} {{firstNameMale}}-{{lastName}}',
+        '{{firstNameMale}}-{{lastName}} {{firstNameMale}}',
+    );
+
+    protected static $femaleNameFormats = array(
+        '{{lastName}} {{firstNameFemale}}',
+        '{{lastName}} {{firstNameFemale}}',
+        '{{lastName}} {{firstNameFemale}}',
+        '{{lastName}} {{firstNameFemale}}',
+        '{{lastName}} {{firstNameFemale}}',
+        '{{firstNameFemale}} {{prefix}} {{lastName}}',
+        '{{lastName}} {{firstNameFemale}}-{{lastName}}',
+        '{{firstNameFemale}}-{{lastName}} {{firstNameFemale}}',
+    );
+
+    protected static $firstNameMale = array(
```

Comment:
For most arrays  there are not official datasource in Côte d'Ivoire, there are based on my experience. what should I do in this case.